react/package.json

```diff
@@ -13,19 +13,24 @@
     "font-awesome": "^4.7.0",
     "graphql-tag": "^2.10.0",
     "hoist-non-react-statics": "^2.5.5",
+    "jest-websocket-mock": "^2.0.0",
     "jsonschema": "^1.2.4",
-    "prop-types": "^15.6.1",
+    "mock-socket": "^9.0.2",
```

Comment:
please remove if not used

Reply:
This is a peer dependency for [jest-websocket-mock](https://github.com/romgain/jest-websocket-mock#readme).
Moved to `devDependencies` alongside with jest-websocket-mock